vizro-ai/changelog.d/new_fragment.md.j2

```diff
@@ -8,7 +8,7 @@ Uncomment the section that is right (remove the HTML comment wrapper).
 <!--
 ### {{ cat }}
 
-- A bullet item for the {{ cat }} category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX. ([#1](https://github.com/mckinsey/vizro/pull/1))
+- A bullet item for the {{ cat }} category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
```

Comment:
Do you know why it's removing the dot in the end? This would make @petar-qb and @antonymilne very sad

Reply:
TBH, I'm happy to exclude the changelog folder from the checking to avoid making anyone 😢  so I'll adjust this

Reply:
This isn't a real change, it's just because we need to merge in `main` fully. So we don't need to exclude changelog after all 🙂 And the `.` can stay!